src/workers/background-proc/core.ts

```diff
@@ -124,6 +131,68 @@ export class BackgroundCore {
     await queue.onIdle()
   }
 
+  async getVideoSize(url: string): Promise<VideoSize> {
+    const text = await Axios.get(url);
+    const textData = text.data;
+    console.log(textData);
+    let regex = /RESOLUTION=(.+),/g;
+    let found = textData.match(regex);
+    if (found === null || found.length === 0) {
+      regex = /RESOLUTION=(.+)\n/g;
+      found = textData.match(regex);
+    }
+    const size = found[0].replace('RESOLUTION=', '').replace(',', '').replace('\n', '');
+    console.log(`Size ${size}`);
+    const sizeComps = size.split(`x`);
+    return {
+      width: parseInt(sizeComps[0]),
+      height: parseInt(sizeComps[1]),
+    }
+  }
+
+  async updateHeightWeight() {
+    const PQueue = (await import('p-queue')).default
+    const queue = new PQueue({ concurrency: 50 })
+
+    // Step 1. Find all posts having 3speak as app name and height-width is null (not updated)
+    const items = await this.posts.find({
+      parent_author: "",
+      "json_metadata.app": {
+        '$regex': '3speak'
+      }, 
+      "json_metadata.video.info.height": { $eq: null },
```

Comment:
Do app_metadata, json_metadata is used for unparsed fields and should be unmodified. 
Example of what app_metadata should look like with spkvideo present
![image](https://github.com/spknetwork/union-indexer-node/assets/47548474/04dc459b-6905-43b6-b13c-ee320c084b65)